helm/prometheus-rules/templates/platform/atlas/alerting-rules/mimir-to-grafana-cloud-exporter.rules.yml

```diff
@@ -0,0 +1,70 @@
+{{- if .Values.mimir.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: mimir-to-grafana-cloud-exporter.rules
+  namespace: {{ .Values.namespace }}
+spec:
+  groups:
+  - name: mimir-to-grafana-cloud-exporter
+    rules:
+    - alert: MimirToGrafanaCloudExporterDown
+      annotations:
+        description: '{{`Prometheus Mimir to Grafana-Cloud is down.`}}'
+        opsrecipe: mimir-grafana-cloud-exporter-failing/
+        dashboard: iWowmlSmk/prometheus?var-cluster=mimir-to-grafana-cloud
+      # We can use absent function here because the prometheus mimir-to-grafana-cloud is a MC component only
+      expr: up{job="mimir/mimir-to-grafana-cloud"} == 0 or absent(up{job="mimir/mimir-to-grafana-cloud"})
```

Comment:
I added `absent` for our specific use case because we know it's related to the MC only.